_posts/2017-03-05-artsy-technology-stack-2017.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: Artsy's Technology Stack, 2017
+date: 2017-03-05
+categories: [Technology, eigen, force, gravity]
+author: orta
+series: Artsy Tech Stack
+---
+
+Artsy has now grown past 160 team members and our Engineering organization is now 30? strong. [WIP] For a brief overview of what the company has accomplished in the last few years, check out our [2013](http://2013.artsy.net) and [2014](http://2014.artsy.net) reviews.
```

Comment:
We're 29 total engineers, including 4 leads, 3 directors and a CTO.

How's this? Needs links all over the place.

_Artsy was launched in 2012 as the Art Genome Project. By 2014 we had 230,000 works of art from 600 museums and institutions and launched our first business, a subscription service for commercial galleries with over 80,000 works for sale and partnerships with 37 art fairs and a handful of benefit auctions. That year collectors from 82 countries inquired on over $5.5B of art. By 2015 we doubled our for sale inventory and aggregated 4,000 of the world's leading galleries and 60 art fairs. We also launched two new businesses: commercial auctions and online media. Lastly, in 2016 we doubled our paid gallery network size to become the largest gallery network in the world and grew to become the most-read online art publication in the world as our highly engaging editorial traffic ballooned 320% and as we launched a live auction and a consignments service with the major auction houses._